### DISCUSSION TITLE
Is it possible to remember calculated folders size for jumping in/out of them (to keep them sorted by size)?

### DISCUSSION THREAD

**Discussion by FilthyChef** [doublecmd/doublecmd#1169] [General]
For example, I'd like to calculate all folder sizes inside of `/usr/share`, sort them by size, and open some of the largest to see what's taking so much space and calculate there as well without affecting previous calculations. Is it currently possible? If not, I'd like to make a **Feature Request**:

Calculated folder sizes are remembered as long user stays within the top folder, allowing for as many jumps to lower folders as he wants and making new size calculations of sub-folders without resetting previous calculations. Calculations only reset in 2 cases:
1) User goes up the folder in which calculations were performed (`/usr/share` -> `/usr`, or `/usr/share` -> `/home`, etc.), or user creates new tab and closes the old one.
2) In size-calculated folder, user performs any operation (delete, duplicate, create, ..) that would affect the calculated size. Though instead of resetting that folder's calculation, it'll append "`?`" at the end of displayed size. Requesting size calculation (pressing Space) on that folder will re-calculate.

**Comment by goyalyashpal** [comment]
Thumbs for need for solution to this problem.

If this is too much to tackle, then a Minimum Viable Proposal can be to remember the directory size calculations of just the unmodified parent directories. Even minimal: remember for only say upto last 3 calculations.

I saw a directory whose calculation was reset to `<DIR>` on some modification (don't remember how i did the modification). but it's position stayed intact. So, the `append ?` thing won't be needed for this use case either.

----
**Verbose details in context with use case:**

My use case was: i was cleaning up the directories (3-5 features worked in tandem)
&nbsp;&nbsp;&nbsp; (1.) using <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>enter</kbd> ("Sho**w** Occupied Space") 
&nbsp;&nbsp;&nbsp; (2.) thanks for fixing https://github.com/doublecmd/doublecmd/issues/806
&nbsp;&nbsp;&nbsp; sorting as largest to smallest with (3.) default automatic refresh on completion of calculation

So, i would enter the top (largest) directory, see the stuff there, and come back just a jiffy later. Even this required recalculation. Modifying the children was not too frequent.

suggested MVP will help a lot in this.
